drivers/source/SerialBase.cpp

```diff
@@ -18,6 +18,8 @@
 #include "platform/mbed_wait_api.h"
 #include "platform/mbed_critical.h"
 #include "platform/mbed_power_mgmt.h"
+#include "mbed_chrono.h"
+#include "ThisThread.h"
```

Comment:
lets use explicit paths here as the above headers (platform for chrono, rtos for thisthread)

Reply:
fixed with bdb44e2b54d21e1b8e552b29a4f3d37098fa4ab8